src/Resources/contao/dca/tl_settings.php

```diff
@@ -164,14 +164,6 @@
 			'inputType'               => 'text',
 			'eval'                    => array('rgxp'=>'url', 'tl_class'=>'w50')
 		),
-		'cacheMode' => array
```

Comment:
Do we need an upgrade task in the install tool that will update all entries in `tl_page` if this setting was set to anything other than `none`?